node/resolvers/Routes/index.ts

```diff
@@ -395,6 +396,16 @@ export const Routes = {
     stateRegistration =
       costCenterResponse.data.getCostCenterById.stateRegistration
 
+    // Only require CPF if cost center contains an address in Brazil
+    // This is a workaround to avoid setting CPF as documentType for countries other than Brazil
+    if (
+      costCenterResponse.data.getCostCenterById.addresses.some(
```

Comment:
Just a minor thing: I believe it's not guaranteed that this `costCenterResponse ` field and children are set, since there's a check further below this file for `costCenterResponse?.data?.getCostCenterById?.addresses?.length` (this `?.` operator basically checks if the variable exists and returns null/undefined instead of accessing the property in that case).

So it'd be safer to add this `?.` operator here too:

```suggestion
      costCenterResponse?.data?.getCostCenterById?.addresses.some(
```

Reply:
That sounds good! Done!